README.md

```diff
@@ -167,6 +167,19 @@ and run tests with
 
 For more information on the integration tests, run `test_integration --help`.
 
+## Building the templates
+
+1. Create a fedora-28 AppVM for building
+2. Increase the disk size to at least 15GB (as the build uses over 10GB)
+3. Import the QubesOS master key and the GPG key used to sign tags (see https://www.qubes-os.org/security/verifying-signatures/)
```

Comment:
We don't actually need this step anymore, given recent script changes, but it's OK to document for a while.